tauri-app/src-tauri/src/seventh_heaven.rs

```diff
@@ -52,7 +52,7 @@ pub(crate) async fn install_run(app_handle: AppHandle) -> Result<(), ()> {
         // TODO - error handling
     }).unwrap();
 
-    let _steam = SteamManager::new(steam_home);
+    let _steam = SteamManager::new(steam_home.clone());
```

Comment:
This is the only way I could figure out how to use this variable on line 69 without making some egregious changes that I didn't really understand, so I hope it's fine